src/swarm/neo/node/ConnectionHandler.d

```diff
@@ -207,19 +146,11 @@ class ConnectionHandler : IConnectionHandler
 
         public void initStats ( RequestStats request_stats )
         {
-            foreach ( code, rq; this.map )
-                if ( rq.name.length > 0 )
-                    request_stats.init(rq.name, rq.timing);
-
             foreach ( command, rq; this.request_info )
```

Comment:
Ooh, I think I just spotted a mistake here. `request_stats.init` only allows each request to be initialised once. `this.request_info` will contain one entry per _version_ of a request (i.e. potentially more than one entry per request).

Reply:
I'll make a PR against v4.x.x.

Reply:
https://github.com/sociomantic-tsunami/swarm/pull/257

Reply:
Yeah, I'll think it will be fine we rebase v5.x.x on top of v4.x.x after that's merged.